frontstage/__init__.py

```diff
@@ -15,12 +16,14 @@
 app_config = 'config.{}'.format(os.environ.get('APP_SETTINGS', 'Config'))
 app.config.from_object(app_config)
 
+log_level = 'DEBUG' if app.config['DEBUG'] else None
+logger_initial_config(service_name='ras-frontstage', log_level=log_level)
 logger = wrap_logger(logging.getLogger(__name__))
+logger.debug('{} used'.format(str(app_config)))
 
-logger.error('{} used'.format(str(app_config)))
 for var in app.config['NON_DEFAULT_VARIABLES']:
```

Comment:
Suggestion:

```
missing = [app.config.get(var) for var in app.config['NON_DEFAULT_VARIABLES'] if app.config.get(var)]
if missing:
    raise MissingEnvironmentVariables(missing, ...)
```
Reports all missing vars in the event there's more than one.

Reply:
Longer-term, we might want to avoid doing this check in the `__init__.py`, as a) it can prevent import of the package which might have unintended consequences and b) it's implicit.